src/pages/home/report/ReportActionItem.js

```diff
@@ -124,8 +124,8 @@ class ReportActionItem extends Component {
      * @param {Object} [event] - A press event.
      */
     showPopover(event) {
-        // Block menu on the message being Edited
-        if (this.props.draftMessage) {
+        // Block menu on the message being Edited or if the report item has errors
```

Comment:
```suggestion
        // Block menu on the message being Edited or if the report action item has errors
```
NAB: Just to be clear we are blocking the report action errors not the report errors